filters/auth/grantlogout.go

```diff
@@ -0,0 +1,192 @@
+//
+// grantLogout filter
+//
+// Used for revoking access/refresh tokens generated by the oauthGrant() filter.
+//
```

Comment:
I think we do not need this doc. We have it in docs.go, which is the package documentation and for users in the docs/*/*.md

Reply:
Removed ✅